spec/ddtrace/contrib/rails/support/base.rb

```diff
@@ -32,6 +33,7 @@
   end
 
   let(:initialize_block) do
+    $stdout.sync = true
```

Comment:
Do you know why this is needed?

I ask because this is a global change that will outlive this test execution, and has impact on other subsequent tests.